gapis/api/gles/externs.go

```diff
@@ -61,15 +61,15 @@ func (e externs) unmapMemory(slice memory.Slice) {
 }
 
 func (e externs) GetEGLStaticContextState(EGLDisplay, EGLContext) StaticContextStateʳ {
-	return FindStaticContextState(e.s.Arena, e.cmd.Extras()).Clone(e.s.Arena, api.CloneContext{})
+	return FindStaticContextState(e.s.Arena, e.cmd.Extras())
```

Comment:
Not sure about this change. "No need to clone what is already a copy." Where is this deep copy of which you speak?

Reply:
Done.